api/src/reporter/reporter.utils.ts

```diff
@@ -1,10 +1,33 @@
-export function flattenReporter(L) {
+import { createCipheriv, createDecipheriv, randomBytes, scrypt } from 'crypto'
+import { promisify } from 'util'
+
+export async function flattenReporter(L) {
   return {
     id: L?.id,
     address: L?.address,
-    privateKey: L?.privateKey,
+    privateKey: await decryptText(L?.privateKey),
     oracleAddress: L?.oracleAddress,
     service: L?.service.name,
     chain: L?.chain?.name
   }
 }
+export async function encryptText(textToEncrypt: string) {
+  const password = process.env.ENCRYPT_PASSWORD || 'bisonai@123'
```

Comment:
@KelvinThai I think we should not have any default password here. If the `ENCRYPT_PASSWORD` is empty, then we should not allow to run neither encryption or decryption process, but detect it right after the launch of the service.